openpype/hosts/blender/plugins/publish/collect_review.py

```diff
@@ -40,13 +42,18 @@ def process(self, instance):
 
             task = instance.context.data["task"]
 
+            # Store focal length in `burninDataMembers`
+            burninDataMembers = instance.data.get("burninDataMembers", {})
```

Comment:
```suggestion
            burninDataMembers = instance.data.setdefault("burninDataMembers", {})
```
If we use this, we dont need line 56 as it will insert the data if it doesn't exist.